src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { AppBar, Box, Toolbar, Typography } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+const NavigationBar = () => {
+  const navigate = useNavigate();
+  return (
+    <Box sx={{ flexGrow: 1 }}>
+      <AppBar
+        position="static"
+        style={{
+          background: "#7a1e77",
```

Comment:
This is the color code for Teal: "#008080". I think you made a typo!

Reply:
We actually discussed this and made an executive decision not to change to teal. 

Purple is the color of royalty after all !

![purp](https://user-images.githubusercontent.com/92157834/152996335-e7bb5c9d-3010-4554-b9ac-5e7f52e4e424.jpeg)

Reply:
That picture is convincing, I submit to the obvious!